### PR TITLE
R 2.20 Feature/faq dcc replacement

### PR DESCRIPTION
**Pull request by dsarkar**

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -543,6 +543,15 @@
                                     "Sollte die Corona-Warn-App durch diese Anpassung etwas an der Gruppierung Ihrer Zertifikate verändern, werden Sie darüber mit einer Benachrichtigung informiert.",
                                     "Weitere Informationen finden Sie im Blog: <a href='../../blog/2022-03-16-cwa-2-19/' target='_blank'>CWA 2.19 verbessert die Zuordnung von Zertifikaten zu Personen</a>"
                                 ]
+                            },
+                            {
+                                "title": "Können Zertifikate über die App aktualisieren werden?",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:548]: "aktualisieren" => "aktualisiert"

File: src/data/faq_de.json [hunk 2]
```diff
@@ -543,6 +543,15 @@
...
+                                "anchor": "dcc_replacement",
+                                "active": false,
+                                "textblock": [
+                                    "Die Spezifikationen der EU für Impfzertifikate für Auffrischimpfungen wurden geändert. Sollte Ihr Zertifikat nicht mehr den aktuellen Spezifikationen entsprechen, ist es zwar weiterhin gültig, es kann jedoch sein, dass bei einer Prüfung die Auffrischimpfung nicht als solche erkannt wird. In solch einem Fall können Sie über Corona-Warn-App ab Version 2.20 mit einer entsprechenden Nachricht darüber informiert werden. Sie können dann freiwillig ein neues Impfzertifikat kostenlos über die App anfordern. Nachdem Sie Ihr Einverständnis gegeben haben, übermittelt die App das Impfzertifikat zur Überprüfung an das RKI, welches ein aktualisiertes Zertifikat ausstellt und an Ihre App übermittelt. Sie finden das aktualisierte Zertifikat dann in der Zertifikate-Registerkarte. Das alte Zertifikat wird automatisch in den Papierkorb verschoben und nach 30 Tagen gelöscht. Das alte Zertifikat kann innerhalb der 30 Tage jederzeit wiederhergestellt werden.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:552]: "über Corona-Warn-App" => "über die Corona-Warn-App" ?

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Suggestions made

File: src/data/faq.json [hunk 3]
```diff
@@ -543,6 +543,15 @@
...
+                            },
+                            {
+                                "title": "Can certificates be updated via the app?",
+                                "anchor": "dcc_replacement",
+                                "active": false,
+                                "textblock": [
+                                    "The EU specifications for booster vaccination certificates have changed. If your certificate no longer meets the current specifications, it is still valid, but it may not be recognized as a booster vaccination during an inspection. In such a case, you can be informed about it via the Corona-Warn-App version 2.20 and later with an appropriate message. You can then voluntarily request a new vaccination certificate free of charge via the app. After you have given your consent, the app transmits the vaccination certificate to the RKI for verification, which issues an updated certificate and transmits it to your app. You will then find the updated certificate in the certificate tab. The old certificate is automatically moved to the bin and deleted after 30 days. The old certificate can be restored at any time within the 30 days.",
```
Comment (Ein-Tim) [inline src/data/faq.json:552]: Won't version 2.19 also support this?

Comment (dsarkar) [reply src/data/faq.json:552]: true ...

**Comment by svengabr** [comment]
Thanks, @Ein-Tim @MikeMcC399 !

I changed the texts accordingly.

**Review by svengabr — APPROVED** [review]

**Pull request merged**